src/jobs/models/jobManager.js

```diff
@@ -8,7 +8,7 @@ const logger = require('../../common/logger'),
     databaseConnector = require('./database/databaseConnector');
 
 let cronJobs = {};
-const JOB_PLATFORM_NAME = 'predator.%s';
+const JOB_PLATFORM_NAME = 'predator-runner.%s';
```

Comment:
what do you say renaming this variable to `JOB_NAME` ? it represents the job name not the platform